src/jgikbase/test/idmapping/storage/mongo/test_id_mapping_mongo_storage.py

```diff
@@ -168,9 +166,9 @@ def test_startup_with_2_config_docs(mongo):
 
     # pattern matcher for the error format across python 2 & 3
     p = re.compile(
-        "Failed to create index: E11000 duplicate key error (index|collection): "
-        + r"test_id_mapping.config( index: |\.\$)schema_1\s+dup key: "
-        + r'\{ : "schema" \}'
+        r'Failed to create index: .*?'
+        r'E11000 duplicate key error collection: .*?'
+        r' index: schema_1 dup key: .*?'
```

Comment:
This no longer tests that the db and collection names are in the error message

Same for the duplicate key

Reply:
👍

Reply:
:+1: